vxtelegram/tests/test_telegram.py

```diff
@@ -276,14 +276,59 @@ def test_translate_inbound_message_from_user(self):
         self.assertEqual(inbound_msg['message_id'], message['telegram_id'])
 
     @inlineCallbacks
+    def test_update_lifetime(self):
+        """
+        update_ids in Redis should expire after update_lifetime has elapsed,
+        meaning they should no longer be considered duplicates
+        """
+        # This test is very pedantic about the value used for update_lifetime,
+        # as well as how long the thread should sleep - change at your own risk
+        transport = yield self.get_transport(update_lifetime=1)
+        yield transport.mark_as_seen(1234)
+        a = yield transport.is_duplicate(1234)
+        self.assertTrue(a)
+
+        # Wait for update_id to expire
+        from time import sleep
+        sleep(2)
```

Comment:
We should try not to call `sleep` in tests for two reasons:
* it blocks the reactor from running
* it makes the test run longer
Instead we can access redis directly (via `self.transport.redis`) and check that the expiry time is set correctly (i.e. `<= 1`).